packages/headless/src/components/WPHead.tsx

```diff
@@ -17,9 +24,25 @@ export default function WPHead(): JSX.Element {
     title = `${siteTitle} – ${siteTagline}`;
   }
 
+  if (post?.enqueuedStylesheets?.nodes) {
+    stylesheet = post.enqueuedStylesheets.nodes
+      .filter((node) => {
+        return node.handle === 'wp-block-library';
```

Comment:
Shouldn't we include all enqueued stylesheets in case other blocks enqueue additional styles?

I know that disabling the theme in some capacity is planned, but as a stop-gap, perhaps we could filter out any URLs that start with `wp-content/themes`? 🤔

Reply:
Updated the filter loop include everything except theme stylesheets.